torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -329,6 +331,12 @@ class TORCH_API ProcessGroupNCCL : public Backend {
     std::vector<at::Tensor> result() override;
 
    protected:
+    // The process group name
+    std::string pgName_;
```

Comment:
How about using name `pgUID` to keep consistent with `ProcessGroupNCCL` calss? Seems its previous name is `pg_name_`, but is changed to `pg_uid_` now.

Reply:
fixed.